src/contracts/middleware/Middleware.sol

```diff
@@ -226,9 +224,7 @@ contract Middleware is
     function setForwarder(
         address forwarder
     ) external checkAccess {
-        if (forwarder == address(0)) {
-            revert Middleware__InvalidAddress();
-        }
+        _checkNotZeroAddress(forwarder);
```

Comment:
Use the modifier.
I only used the internal method directly in the initialize because of stack too deep.
But actually now that it's a struct we might also use it there to further reduce the size.